logging/src/main/java/se/fortnox/reactivewizard/logging/LoggingFactory.java

```diff
@@ -54,6 +63,22 @@ public void init() {
             props.setProperty("log4j.logger." + e.getKey(), e.getValue());
         }
         LoggingConfiguration.getInstance().configure(props);
+
+        // Provides a facility to reconfigure JUL via a reactivewizard config.
+        if (julLoggingActivation) {
+            configureJavaUtilLogging();
+        }
+    }
+
+    private void configureJavaUtilLogging() {
+        final InputStream stream = LoggingFactory.class.getClassLoader().getResourceAsStream("rw-jul-logging.properties");
+        if (stream != null) {
+            try {
+                getLogManager().readConfiguration(stream);
+            } catch (IOException e) {
+                e.printStackTrace();
```

Comment:
We probably want to throw this to avoid application from starting without correct logging?